cds_ils/importer/providers/cds/rules/base.py

```diff
@@ -406,47 +406,56 @@ def open_access(self, key, value):
     raise IgnoreKey("_migration")
 
 
-@model.over("urls", "^8564_")
+@model.over("urls", "(^8564_)|(^8654_)")
 @filter_list_values
 @for_each_value
 def urls(self, key, value):
     """Translates urls field."""
-    sub_y = clean_val("y", value, str, default="")
-    sub_u = clean_val("u", value, str, req=True)
-
-    eitems_ebl = self["_migration"]["eitems_ebl"]
-    eitems_external = self["_migration"]["eitems_external"]
-    eitems_proxy = self["_migration"]["eitems_proxy"]
-    eitems_files = self["_migration"]["eitems_file_links"]
-
-    url = {"value": sub_u}
-    if sub_y and sub_y != "ebook":
-        url["description"] = sub_y
-
-    # EBL publisher login required
-    if all([elem in sub_u for elem in ["cds", ".cern.ch" "/auth.py"]]):
-        eitems_ebl.append(url)
-        self["_migration"]["eitems_has_ebl"] = True
-    # EzProxy links
-    elif "ezproxy.cern.ch" in sub_u:
-        url["value"] = url["value"].replace(
-            "https://ezproxy.cern.ch/login?url=", ""
-        )
-        eitems_proxy.append(url)
-        self["_migration"]["eitems_has_proxy"] = True
-    # local files
-    elif all(
-        [elem in sub_u for elem in ["cds", ".cern.ch/record/", "/files"]]
-    ):
-        eitems_files.append(url)
-        self["_migration"]["eitems_has_files"] = True
-    elif sub_y == "ebook" or sub_y == "e-proceedings":
-        eitems_external.append(url)
-        self["_migration"]["eitems_has_external"] = True
-    else:
-        # if none of the above, it is just external url
-        # attached to the document
-        return url
+    if key == "8654_":
+        url_dict = {
+            "value": clean_val("u", value, str),
+            "description": clean_val("y", value, str),
+        }
+        return url_dict
```

Comment:
in this case I would split those rules, since you return anyway, and we will not have additional level of if statements.